domainobjs/ts/__tests__/DomainObjs.test.ts

```diff
@@ -1,286 +0,0 @@
-            const unserialized = StateLeaf.unserialize(serialized);
-
-            expect(unserialized.voiceCreditBalance.toString()).to.eq(
-                stateLeaf.voiceCreditBalance.toString()
-            );
-        });
-    });
-
-    describe("Keypairs", () => {
-        it("the Keypair constructor should generate a random keypair if not provided a private key", () => {
-            const k1 = new Keypair();
-            const k2 = new Keypair();
-
-            expect(k1.equals(k2)).to.be.false;
-
-            expect(k1.privKey.rawPrivKey).not.to.eq(k2.privKey.rawPrivKey);
-        });
-
-        it("the Keypair constructor should generate the correct public key given a private key", () => {
-            const rawKeyPair = genKeypair();
-            const k = new Keypair(new PrivKey(rawKeyPair.privKey));
-            expect(rawKeyPair.pubKey[0]).to.eq(k.pubKey.rawPubKey[0]);
-            expect(rawKeyPair.pubKey[1]).to.eq(k.pubKey.rawPubKey[1]);
-        });
-
-        it("PrivKey.serialize() and unserialize() should work correctly", () => {
-            const k = new Keypair();
-            const sk1 = k.privKey;
-
-            const s = sk1.serialize();
-            expect(s.startsWith("macisk.")).to.be.true;
-
-            const d = "0x" + s.slice(7);
-            expect(sk1.rawPrivKey.toString()).to.eq(BigInt(d).toString());
-
-            const c = PrivKey.unserialize(s);
-            expect(sk1.rawPrivKey.toString()).to.eq(
-                BigInt(`${c.rawPrivKey}`).toString()
-            );
-        });
-
-        it("PrivKey.isValidSerializedPrivKey() should work correctly", () => {
-            const k = new Keypair();
-            const s = k.privKey.serialize();
-
-            expect(PrivKey.isValidSerializedPrivKey(s)).to.be.true;
-            expect(PrivKey.isValidSerializedPrivKey(s.slice(1))).to.be.false;
-        });
-
-        it("PubKey.isValidSerializedPubKey() should work correctly", () => {
-            const k = new Keypair();
-            const s = k.pubKey.serialize();
-
-            expect(PubKey.isValidSerializedPubKey(s)).to.be.true;
-            expect(
-                PubKey.isValidSerializedPubKey(
-                    s + "ffffffffffffffffffffffffffffff"
-                )
-            ).to.be.false;
-            expect(PubKey.isValidSerializedPubKey(s.slice(1))).to.be.false;
-        });
-
-        it("PubKey.serialize() and unserialize() should work correctly", () => {
-            const k = new Keypair();
-            const pk1 = k.pubKey;
-
-            const s = pk1.serialize();
-            expect(s.startsWith("macipk.")).to.be.true;
-
-            const d = s.slice(7);
-            const unpacked = unpackPubKey(Buffer.from(d, "hex"));
-
-            expect(unpacked[0].toString()).to.eq(pk1.rawPubKey[0].toString());
-            expect(unpacked[1].toString()).to.eq(pk1.rawPubKey[1].toString());
-        });
-
-        it("PrivKey.copy() should produce a deep copy", () => {
```

Comment:
This test looks to be missing from`privateKey.test.ts`

Reply:
added back, good catch